capa/render/default.py

```diff
@@ -148,7 +152,47 @@ def render_attack(doc: rd.ResultDocument, ostream: StringIO):
     if rows:
         ostream.write(
             tabulate.tabulate(
-                rows, headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)], tablefmt="mixed_grid"
+                rows,
+                headers=[width("ATT&CK Tactic", 20), width("ATT&CK Technique", 80)],
+                tablefmt="mixed_grid",
+            )
+        )
+        ostream.write("\n")
+
+
+def render_maec(doc: rd.ResultDocument, ostream: StringIO):
+    """
+    example::
+
+        +--------------------------+-----------------------------------------------------------+
+        | MAEC Category            | MAEC Value                                                |
+        |--------------------------+-----------------------------------------------------------|
+        | analysis-conclusion      | malicious                                                 |
+        |--------------------------+-----------------------------------------------------------|
+        | malware-family           | PlugX                                                     |
+        |--------------------------+-----------------------------------------------------------|
+        | malware-category         | downloader                                                |
+        |                          | launcher                                                  |
+        +--------------------------+-----------------------------------------------------------+
+    """
+    maec_schema = collections.defaultdict(set)
+    for rule in rutils.maec_rules(doc):
+        for category, value in rule.meta.maec:
+            if value:
+                maec_category = str(rule.meta.maec.__fields__[str(category)].alias)
```

Comment:
this appears hacky, is there a better way?

Reply:
Indeed it is :)

I couldn't figure out a simpler way, that's why I initially used the `if` block.

Reply:
ok, let's use a hard-coded set with the keys "analysis-conclusion", "malware-family", etc. -- that way we also control the order reliably